eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -32,6 +32,16 @@ jobs:
           DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
           DOTNET_CLI_TELEMETRY_OPTOUT: 1
           DOTNET_MULTILEVEL_LOOKUP: 0
+      - task: PowerShell@2
+        displayName: "Build Sample Apps"
+        env:
+          DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+          DOTNET_CLI_TELEMETRY_OPTOUT: 1
+          DOTNET_MULTILEVEL_LOOKUP: 0
+        inputs:
+          filePath: "eng/BuildSamples.ps1"
```

Comment:
Do we really want to build all samples with every pipeline? I'd suggest only building the samples for the given service directory and perhaps any root level samples.

Reply:
Or if you goal is to only do it as part of the core build then you will need to add these steps in a more localized way to the core/ci.yml file instead of in the common template that happens for every pipeline.

Reply:
The samples are specific to individual services as they are sample applications that will generally span multiple services.